cpp/ql/test/duplication-tests/constants/constants.cpp

```diff
@@ -0,0 +1,12 @@
+
+int x = int();
+float y = float();
+double z = double();
+
+struct R {};
+struct S {
+  S() : S(R()) { }
+  S(R) { }
+};
+/* This produces a getValueText() of 0 which is debatable.  */
```

Comment:
What is "This"? I don't see anything in `expr.expected` after line 8.

Reply:
I was referring to the expression that gets built at line 8 for S(R()).

Reply:
I'll tidy up the comment.